style.css

```diff
@@ -0,0 +1,32 @@
+
+nav{
+    margin-bottom: 5rem !important;
+}
+
+nav ul a{
+
+    color: black !important;
+    font-weight: 500;
+    font-size: large;
+    margin-left: 5em !important;
+}
+
+
+footer{
+    background-color: rgb(204, 197, 197);
+    display: flex;
+    flex-direction: column;
+}
+
+footer .container{
+    display: flex;
+}
+
+
+
+
+@media only screen and (max-width: 600px) {
+    body {
+      background-color: lightblue;
+    }
+  }
```

Comment:
This block of codes is not working. please check and fix it.

Reply:
- [ ]  Good code reviews also help the student to learn. You can point out why you think this code is not working and include links to resources that the student can use to fix their code.

Reply:
- [ ] Sounding politely is key in every code review. Please make it a practice to always sound polite.